flink/v1.17/flink/src/main/java/org/apache/iceberg/flink/sink/ManifestOutputFileFactory.java

```diff
@@ -70,7 +65,10 @@ private String generatePath(long checkpointId) {
   }
 
   OutputFile create(long checkpointId) {
-    String flinkManifestDir = props.get(FLINK_MANIFEST_LOCATION);
+    Table table = tableSupplier.get();
+    TableOperations ops = ((HasTableOperations) table).operations();
+
+    String flinkManifestDir = table.properties().get(FLINK_MANIFEST_LOCATION);
```

Comment:
What happens, if someone commits a change which sets this property? Do we store absolute path for the manifests? Or we try to recalculate, and potentially fail?

Reply:
I reverted this so the properties are stored as a member variable as before.

Reply:
One note, this will reset the property if the task restarts (same as before).